.compose.env.example

```diff
@@ -1,7 +1,7 @@
-# Determines how the dev environment should run. 
+# Determines how the dev environment should run.
```

Comment:
get rid of these changes to `.compose.env.example`

Reply:
@awcrosby not sure how to reverse these changes.. seems like these lines were removed and added back